container/controller.go

```diff
@@ -278,8 +278,12 @@ func NewController(options ControllerOptions) (*Controller, error) {
 	// Load keys
 	keyshutdown := make(chan interface{})
 	go func() {
-		<-c.closing
+		// This is a ridiculous shutdown pattern I'll follow for the sake of
+		// not destabilizing the controller this late in a release.
+		// TODO: Make this not stupid
+		errc := <-c.closing
```

Comment:
this PR might step on this change: 
https://github.com/control-center/serviced/pull/2996